crates/db/types/src/traits.rs

```diff
@@ -359,11 +350,67 @@ pub trait L1BroadcastDatabase: Send + Sync + 'static {
     fn get_last_tx_entry(&self) -> DbResult<Option<L1TxEntry>>;
 }
 
+// =============================================================================
+// MMR Database Traits
+// =============================================================================
+
+/// Unified MMR database trait using MmrId for identification
+///
+/// This trait provides a simpler, more direct API where the MMR instance
+/// is identified by an `MmrId` enum rather than a generic scope parameter.
+/// This is the preferred interface for new code.
+pub trait UnifiedMmrDatabase: Send + Sync + 'static {
+    /// Append a new leaf to the specified MMR
+    fn append_leaf(&self, mmr_id: MmrId, hash: [u8; 32]) -> DbResult<u64>;
+
+    /// Get a node hash by MMR position
```

Comment:
nit: I think that "node label" is more common; one may confuse to think that we hash the (already hashed) node label.